seed/challenges/03-front-end-libraries/react.json

```diff
@@ -2155,7 +2155,7 @@
         "If you import styles from a stylesheet, it isn't much different at all. You apply a class to your JSX element using the <code>className</code> attribute, and apply styles to the class in your stylesheet. Another option is to apply <strong><em>inline</em></strong> styles, which are very common in ReactJS development.",
         "You apply inline styles to JSX elements similar to how you do it in HTML, but with a few JSX differences. Here's an example of an inline style in HTML:",
         "<code>&lt;div style=\"color: yellow; font-size: 16px\"&gt;Mellow Yellow&lt;/div&gt;</code>",
-        "JSX elements use the <code>style</code> attribute, but because of the way JSX is transpiled, you can't set the value to a <code>string</code>. Instead, you set it equal to a JavaScript <code>object</code>. Here's an example:",
+        "JSX elements use the <code>style</code> attribute, but because of the way JSX is transpiled, you can't set the value to a <code>string</code>. Instead, you set it equal to a JavaScript <code>object</code>. Notice how we camelCase the \"fontSize\" property, this will automatically become hyphenated once the JSX is transpiled. Here's an example:",
```

Comment:
Notice how we camelCase the \"fontSize\" property, this will ~automatically become hyphenated once the JSX is transpiled.~

Notice how we camelCase the \"fontSize\" property? This is because React will not accept `kebab-case` keys in the style object. React will apply the correct property name for us in the HTML.

Or something to that effect.